src/parsers/chrono/chrono.js

```diff
@@ -56,6 +56,31 @@ dayPeriodsParser.extract = (text, ref, match) => {
   });
 };
 
+const currentMonthDayParser = Object.assign(
+  new chrono.Parser(),
+  {
+    pattern: () => /\b(?:on|by)\s+the\s+(\d\d?)(?:th|st|rd|nd)\b/,
```

Comment:
There is only so much we can do with pattern matching and at the same time maintaining readible code.
I don't think anybody would naturally say 'Remind me to go shopping at the 4th.'